redaxo/src/core/lib/response.php

```diff
@@ -94,6 +94,17 @@ private static function sendPreloadHeaders()
         }
     }
 
+    private static function sendServerTimingHeaders()
+    {
+        if (rex::isDebugMode() || ($user = rex_backend_login::createUser()) && $user->isAdmin()) {
```

Comment:
Hmm ggf. Auch nur rex::getUser() 🤔?

Reply:
```suggestion
        if (rex::isDebugMode() || ($user = rex::getUser()) && $user->isAdmin()) {
```
würde auch funktionieren

Reply:
rex::getUser() ist besser weil nicht versucht wird ein user anzulegen / die session zu starten